cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -168,6 +169,29 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def create_lookup_index() {
+    // remove autogenerated lookup indexes
+    val res = execute("SHOW LOOKUP INDEXES YIELD name")
+    res.columnAs[String]("name").foreach(n => execute(s"DROP INDEX $n"))
+
+    testQuery(
+      title = "Create a lookup index for nodes",
+      text = "A named lookup index for all nodes that have any label can be created with `CREATE LOOKUP INDEX index_name FOR (n) ON EACH labels(n)`. " +
+        "Note that the index is not immediately available, but will be created in the background.",
```

Comment:
```suggestion
        "Note that the index is not immediately available, but is created in the background.",
```

Reply:
I think this is just copied over from the existing btree indexes so then we should probably update there as well (and in the pr for fulltext indexes)

Reply:
Yes, I agree. It's an ongoing process to be consistent with the active voice and present tense.

Reply:
Did not seem to be applicable to fulltext indexes but have updated the btree ones in addition to the lookup